cli/bpmetadata/cmd.go

```diff
@@ -154,20 +182,15 @@ func createInfo(bpPath string, readmeContent []byte) (*BlueprintInfo, error) {
 	}
 
 	versionInfo, err := getBlueprintVersion(path.Join(bpPath, tfVersionsFileName))
-	if err != nil {
-		return nil, err
-	}
-
-	i.Version = versionInfo.moduleVersion
-
-	// actuation tool
-	i.ActuationTool = BlueprintActuationTool{
-		Version: versionInfo.requiredTfVersion,
-		Flavor:  "Terraform",
+	if err == nil {
+		i.Version = versionInfo.moduleVersion
+		i.ActuationTool = BlueprintActuationTool{
+			Version: versionInfo.requiredTfVersion,
+			Flavor:  "Terraform",
```

Comment:
Could we always set `Flavor` since we currently only support TF and there is currently no logic to determine this anyway?

Reply:
This is always setting `Flavor` as `Terraform`. Were you suggesting that we set the default as `Terraform` in the type definition or just take it out altogether?